vue/sbc-common-components/src/components/SbcSignin.vue

```diff
@@ -64,6 +66,10 @@ export default class SbcSignin extends NavigationMixin {
           await KeyCloakService.initSession()
           // tell KeycloakServices to load the user info
           const userInfo = await this.loadUserInfo()
+
+          //update user profile
+          await this.updateUserProfile()
```

Comment:
hope you tested well and made sure this is called only during the signin ...

Reply:
Yea, did some testing.